sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -260,6 +261,8 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
         },
         assessmentTriggerType: application.currentAssessment.triggerType,
         appealStatus: StudentAppealStatus.Pending,
+        applicationOfferingChangeRequestId:
```

Comment:
I believe this API is for the tracker. Why do we need this property for the current story?

Reply:
**Review Changes** button in `Completed.vue` passes the `applicationOfferingChangeRequestId` as a parameter to `viewApplicationOfferingChangeRequest` to enable the appropriate route navigation.